webapp/actions/global_actions.jsx

```diff
@@ -534,3 +538,33 @@ export function emitBrowserFocus(focus) {
         focus
     });
 }
+
+export function redirectUserToDefaultTeam() {
+    const teams = TeamStore.getAll();
+    const teamMembers = TeamStore.getMyTeamMembers();
+    let teamId = BrowserStore.getGlobalItem('team');
+
+    if (!teamId && teamMembers.length > 0) {
+        let myTeams = [];
```

Comment:
this was following the PM spec's and the main idea is to get rid of the team selection page